src/api/BdmsContextExtensions.cs

```diff
@@ -0,0 +1,360 @@
+            .RuleFor(o => o.UscsDetermination, _ => default!)
+            .RuleFor(o => o.DescriptionFacies, f => f.Random.Words(5).OrNull(f, .05f))
+            .RuleFor(o => o.DescriptionLithological, f => f.Random.Words(3).OrNull(f, .05f))
+            .RuleFor(o => o.Import, f => f.Random.Int().OrNull(f, .05f))
+            .RuleFor(o => o.Instrument, f => f.Music.Genre().OrNull(f, .05f))
+            .RuleFor(o => o.LithologyTopBedrockId, f => f.PickRandom(context.Codelists.Where(c => c.Schema == "custom.lithology_top_bedrock").Select(s => s.Id).ToList()).OrNull(f, .05f))
+            .RuleFor(o => o.LithologyTopBedrock, _ => default!)
+            .RuleFor(o => o.Notes, f => f.Random.Words(4).OrNull(f, .05f))
+            .RuleFor(o => o.OriginalUscs, f => f.Random.Word().OrNull(f, .05f))
+            .RuleFor(o => o.UnconrocksId, f => f.PickRandom(context.Codelists.Where(c => c.Schema == "mcla101").Select(s => s.Id).ToList()).OrNull(f, .05f)) // unclear which codelist
+            .RuleFor(o => o.Unconrocks, _ => default!)
+            .RuleFor(o => o.Id, f => layer_ids++);
+
+        Layer SeededLayers(int seed) => fakelayers.UseSeed(seed).Generate();
+
+        for (int i = 0; i < stratigraphyRange.Count(); i++)
+        {
+            // Add 10 layers per stratigraphy
+            var start = (i * 10) + 1;
+            var range = Enumerable.Range(start, 10);  // ints in range must be different on each loop, so that properties are not repeated in dataset.
+            context.Layers.AddRange(range.Select(SeededLayers));
+            context.SaveChanges();
+        }
+
+        // Seed workflows
+        var workflow_ids = 5000;
+        var workflowRange = Enumerable.Range(workflow_ids, 200);
+        var fakeWorkflows = new Faker<Workflow>()
+               .StrictMode(true)
+               .RuleFor(o => o.Id, f => workflow_ids++)
+               .RuleFor(o => o.UserId, f => f.PickRandom(userRange))
+               .RuleFor(o => o.User, _ => default!)
+               .RuleFor(o => o.BoreholeId, f => f.PickRandom(boreholeRange))
+               .RuleFor(o => o.Borehole, _ => default!)
+               .RuleFor(o => o.Notes, f => f.Random.Words(4))
+               .RuleFor(o => o.Mentions, f => new[] { f.Random.Word(), f.Random.Word(), f.Random.Word(), f.Random.Word() })
+               .RuleFor(o => o.Role, f => f.PickRandom<Role>())
+               .RuleFor(o => o.Started, f => f.Date.Between(new DateTime(1990, 1, 1), new DateTime(2005, 1, 1)))
+               .RuleFor(o => o.Finished, f => f.Date.Between(new DateTime(2005, 2, 1), new DateTime(2022, 1, 1)));
+
+        Workflow SeededWorkflows(int seed) => fakeWorkflows.UseSeed(seed).Generate();
+        context.Workflows.AddRange(workflowRange.Select(SeededWorkflows));
+        context.SaveChanges();
+
+        //// Sync all database sequences
```

Comment:
PP: Unnötige Schrägstrich vermeiden
```suggestion
        // Sync all database sequences
```